### PR TITLE
Github Actions

### PR DESCRIPTION
**Pull request by f-squirrel**
The PR introduces `Github Actions CI`.

Main features:
1. `20` concurrent jobs for the Free Plan
2. Job execution time - Each job in a workflow can run for up to 6 hours of execution time
3. Uploading of the build artifacts(logs, core dump files, and built binaries)
4. Ability to log into the host system on which the actual build will run
5. Ability to schedule jobs(another PR with Coverity support is on the way)

### CODE REVIEW

File: test.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+jobs:
+  build:
+    # Job name is Greeting
+    name: Greeting
+    # This job runs on Linux
+    runs-on: ubuntu-latest
+    strategy:
+        matrix:
+            compiler: [gcc, clang]
+            #ci_build_type: ["-DCMAKE_BUILD_TYPE=DEBUG -DCI_TEST_STORAGE_TYPE=v1direct",
```
Comment (andrewjstone) [inline test.yml:22]: Are these comments useful? Can we remove them?

File: tests/simpleTest/persistency_test.cpp [hunk 2]
```diff
@@ -90,6 +90,7 @@ TEST_F(PersistencyTest, PrimaryRestartVC) {
     ISimpleTestReplicaBehavior *b = new OneTimePrimaryDownVC(rp);
     create_and_run_replica(rp, b);
   }
+  // abort();
```
Comment (andrewjstone) [inline tests/simpleTest/persistency_test.cpp:93]: Can we remove this line? I assume it was just for testing failure.

File: to_remove_travis.yml [hunk 3]
```diff
@@ -0,0 +1,43 @@
+os: linux
```
Comment (andrewjstone) [inline to_remove_travis.yml:1]: What is this file for?

Comment (f-squirrel) [reply to_remove_travis.yml]: Removed

File: tests/simpleTest/persistency_test.cpp (hunk 2 above)
Comment (f-squirrel) [reply tests/simpleTest/persistency_test.cpp]: yes, it was for testing. removed

**Pull request merged**